crates/ruff_linter/src/rules/refurb/rules/isinstance_type_none.rs

```diff
@@ -69,7 +68,8 @@ pub(crate) fn isinstance_type_none(checker: &mut Checker, call: &ast::ExprCall)
         return;
     }
 
-    let fix = replace_with_identity_check(expr, call.range, checker);
+    let negate = false;
+    let fix = replace_with_identity_check(expr, call.range, negate, checker);
```

Comment:
Nit: IDE's can show the argument names inline

```suggestion
    let fix = replace_with_identity_check(expr, call.range, false, checker);
```